docs/README.md

```diff
@@ -1,27 +1,22 @@
 # Taurus Data Model Documentation
 
-This the next major data model developed by Citrine, codename `taurus`.
+This is the next major data model developed by Citrine, codename `taurus`.
 
-The format links together materials, the processes that produced them,
-and the measurements that characterize them.
-This facilitates the backwards traversal from a measurement to the material on which it was performed
-to the process by which it was produced to the materials which were used in that process and so on and so forth.
-This generalizes and matures the PIF's `preparation` and `subSystems` objects.
+The model links together materials, the processes that produced them, and the measurements that characterize them.
+This facilitates the backwards traversal from a measurement to the material on which it was performed to the process by which it was produced to the materials which were used in that process. 
+It generalizes and matures `preparation` and `subSystems` objects within the Physical Information File (PIF).
```

Comment:
```suggestion
It generalizes and matures `preparation` and `subSystems` objects within the PIF ([Physical Information File](https://citrineinformatics.github.io/pif-documentation/)).
```

As the document is more likely read by someone familiar with our previous technology, I think the acronym should precede.